### PR TITLE
Add CRDL CLI commands for PulpExporter

### PR DESCRIPTION
**Pull request by ggainey**
closes #7763

### CODE REVIEW

File: pulpcore/cli/core/pulpexport.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
+from typing import List
+import click
+from pulpcore.cli import main
+
+RepoIdentifier = click.Tuple([str, str])
```
Comment (mdellweg) [inline pulpcore/cli/core/pulpexport.py:5]: :+1:

File: pulpcore/cli/core/pulpexport.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+
+@main.group()
+@click.pass_context
+def pulpexporter(ctx):
```
Comment (mdellweg) [inline pulpcore/cli/core/pulpexport.py:10]: I started to adopt a different (more specific) pattern here. See core/task.py and #23 .

File: pulpcore/cli/core/pulpexport.py (hunk 1 above)
Comment (ggainey) [reply pulpcore/cli/core/pulpexport.py:5]: click and mypy and typing are mad at each other, I have work to do.

File: pulpcore/cli/core/pulpexport.py (hunk 2 above)
Comment (ggainey) [reply pulpcore/cli/core/pulpexport.py:10]: Ah interesting - when #23 is merged, I will revisit and follow the trail you're blazing!

**Comment by daviddavis** [comment]
This is great. The one thing I'd suggest is perhaps making a cli group to nest the exporters under (eg `pulp exporter pulp ...`).

**Comment by ggainey** [comment]
I'm doing a major rework of this - I'm going to close this PR, and submit a new one when I'm done.

**Pull request closed without merging**